Added detailed instructions in README with gifs. (#27)
numpy/numpy-tutorials: README.md

### Adding your own tutorials

If you have your own tutorial in the form of a Jupyter notebook (a `.ipynb`
file) and you'd like to try add it out to the repository:

#### Create an issue

Go to [https://github.com/numpy/numpy-tutorials/issues](https://github.com/numpy/numpy-tutorials/issues) and create a new issue with your proposal. Give as much detail as you can about what kind of content you would like to write (tutorial, how-to) and what you plan to cover. We will try to respond as quickly as possible with comments, if applicable.

#### Upload your content

Remember to clear all outputs on your notebook before uploading it. 

> **1. Fork this repository** (if you haven't before)
> <img src="images/01-fork.gif" width=80% height=80%>

Reviewer: Make images collapsible.
```suggestion
<details>
<summary>
**1. Fork this repository** (if you haven't before)
</summary>
<img src="images/01-fork.gif" width=80% height=80%>
<details>
```

Author: Thanks! Sorry for the late reply.

> **2. In your own fork, create a new branch for your content**
> <img src="images/02-create_new_branch.gif" width=80% height=80%>

Reviewer: ```suggestion
<details>
<summary>
**2. In your own fork, create a new branch for your content**
</summary>
<img src="images/02-create_new_branch.gif" width=80% height=80%>
</details>
```

> **3. Add your notebook to the `content/` directory**
> <img src="images/03-upload.gif" width=80% height=80%>

Reviewer: ```suggestion
<details>
<summary>
**3. Add your notebook to the `content/` directory**
</summary>
<img src="images/03-upload.gif" width=80% height=80%>
</details>
```

**4. Update the `environment.yml` file with the dependencies for your tutorial**
(only if you add new dependencies)

> **5. Update this `README.md` to include your new entry**
> <img src="images/04-add_to_readme.gif" width=80% height=80%>

Reviewer: ```suggestion
<details>
<summary>
**5. Update this `README.md` to include your new entry**
</summary>
<img src="images/04-add_to_readme.gif" width=80% height=80%>
</details>
```

**6. Update the attribution section (below) to credit the original tutorial
author, if applicable.**

> **7. Create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)**
> <img src="images/05-create_PR.gif" width=80% height=80%>

Reviewer: ```suggestion
<details>
<summary>
**7. Create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)**
</summary>
<img src="images/05-create_PR.gif" width=80% height=80%>
<details>
```

### Adding your own tutorials

If you have your own tutorial in the form of a Jupyter notebook (a `.ipynb`
file) and you'd like to try add it out to the repository:

#### Create an issue

Go to [https://github.com/numpy/numpy-tutorials/issues](https://github.com/numpy/numpy-tutorials/issues) and create a new issue with your proposal. Give as much detail as you can about what kind of content you would like to write (tutorial, how-to) and what you plan to cover. We will try to respond as quickly as possible with comments, if applicable.

#### Upload your content

> 

Reviewer: ```suggestion
<ul>
```

Remember to clear all outputs on your notebook before uploading it. 

<details>
    <summary>

>         **1. Fork this repository** (if you haven't before)

Reviewer: ```suggestion
        <b>Fork this repository</b> (if you haven't before).
```

    </summary>
    <img src="images/01-fork.gif" width=80% height=80%>
</details>
    
<details>
    <summary>

>         **2. In your own fork, create a new branch for your content**

Reviewer: ```suggestion
        <b>In your own fork, create a new branch for your content.,</b>
```

    </summary>
    <img src="images/02-create_new_branch.gif" width=80% height=80%>
</details>

<details>
    <summary>

>         **3. Add your notebook to the `content/` directory**

Reviewer: ```suggestion
        <b>Add your notebook to the <code>content/</code> directory</b>.
```

    </summary>
    <img src="images/03-upload.gif" width=80% height=80%>
</details>

> **4. Update the `environment.yml` file with the dependencies for your tutorial**

Reviewer: ```suggestion
<b>Update the `environment.yml` file with the dependencies for your tutorial</b>
```

> (only if you add new dependencies)

Reviewer: ```suggestion
(only if you add new dependencies).
```

<details>
    <summary>

>         **5. Update this `README.md` to include your new entry**

Reviewer: ```suggestion
        <b>Update this <code>README.md</code> to include your new entry.</b>
```

    </summary>
    <img src="images/04-add_to_readme.gif" width=80% height=80%>
</details>

> **6. Update the attribution section (below) to credit the original tutorial
> author, if applicable.**

Reviewer: ```suggestion
<b>Update the attribution section (below) to credit the original tutorial
author, if applicable.<b>
```

<details>
    <summary>

>         **7. Create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)**

Reviewer: ```suggestion
        <b>Create a <a href="https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests"> pull request.</a></b>
```

    </summary>
    <img src="images/05-create_PR.gif" width=80% height=80%>
</details>

> :tada: **Wait for review!**

Reviewer: ```suggestion
:tada: <b>Wait for review!</b>
</ul>

```
